meta2v2/meta2_utils_json_in.c

```diff
@@ -79,13 +81,15 @@ m2v2_json_load_single_header (struct json_object *j, gpointer *pbean)
 	GError *err = NULL;
 	GByteArray *id = NULL, *hash = NULL;
 	struct bean_CONTENTS_HEADERS_s *header = NULL;
-	struct json_object *jid, *jhash, *jsize, *jctime, *jmtime, *jmethod, *jtype;
+	struct json_object *jid, *jhash, *jsize, *jctime, *jmtime, *jpolicy,
+			*jmethod, *jtype;
 	struct oio_ext_json_mapping_s mapping[] = {
-		{"id",     &jid,    json_type_string, 1},
-		{"hash",   &jhash,  json_type_string, 1},
-		{"size",   &jsize,  json_type_int, 1},
-		{"ctime",  &jctime, json_type_int, 0},
-		{"mtime",  &jmtime, json_type_int, 0},
+		{"id",           &jid,     json_type_string, 1},
+		{"hash",         &jhash,   json_type_string, 0},
```

Comment:
The hash is no more mandatory, I think you should document why hash is no more mandatory here